Pinta.Core/Classes/BasePaintBrush.cs

```diff
@@ -72,12 +72,10 @@ public virtual void DoMouseDown ()
 
 	public virtual RectangleI DoMouseMove (
```

Comment:
Unrelated, but I just noticed that the public `DoMouseMove|Up|Down` functions are `virtual` which seems wrong, since derived classes are expected to instead implement `OnMouseMove|Up|Down`
Could you remove the virtual from those three methods while you're in here?

Reply:
Sure, I just removed it